syntax = "proto3";

package test;

option go_package = "go-grpc-test/test-proto;test_proto";

service Test {
  rpc ControlGate(stream ControlGateRequest) returns (stream ControlGateResponse) {}
}

message ControlGateRequest {
  string serial = 1;
  string gate_status = 2;
}
message ControlGateResponse {
  string control = 1;
}